spec/integration/support/inspect_and_build_examples.rb

```diff
@@ -26,7 +26,7 @@
       it "inspects" do
         measure("Inspect") do
           @machinery.run_command(
-              "machinery inspect #{@subject_system.ip} -x --name=build_test",
+              "machinery --exclude=/packages/name=test-quote-char inspect #{@subject_system.ip} -x --name=build_test",
```

Comment:
Line is too long. [118/100]<br>Indent the first parameter one step more than the previous line.